vector/src/main/res/layout/item_room.xml

```diff
@@ -165,7 +165,6 @@
         android:visibility="gone"
         app:layout_constraintBottom_toBottomOf="@id/roomNameView"
         app:layout_constraintEnd_toStartOf="@id/roomLastEventTimeView"
-        app:layout_constraintStart_toEndOf="@id/roomDraftBadge"
```

Comment:
was this constraint the cause of the jumping?

Reply:
yes, it was end and start constrained with fixed size, which made the issue.